File: SAM Issue 3/updated SAM files/Java/Q05bCorrect.java
import java.util.Scanner;
public class Q05b 
{
    public static void main(String[] args) 
    {
		// Write your code here	
        Scanner input = new Scanner(System.in);
        int totalSpend = 0;
        System.out.println ("What is your total spend?");
        totalSpend = input.nextInt();
       
        if (totalSpend > 300)
            System.out.println ("Discount is 10%");
        else if (totalSpend > 0)
            System.out.println ("No discount");
        else
            System.out.println ("Invalid input");
    }
}

File: SAM Issue 3/updated SAM files/C#/Q05bCorrect.cs
using System;
namespace Q05b_cs
{
    class Program
    {
        static void Main(string[] args)
        {
			// Write your code here	
            int totalSpend = 0;
            Console.WriteLine("What is your total spend?");
            totalSpend = Convert.ToInt32(Console.ReadLine());
            if (totalSpend > 300)
                Console.WriteLine("Discount is 10%");
            else if (totalSpend > 0)
                Console.WriteLine("No discount");
            else
                Console.WriteLine("Invalid input");
        }
    }
}
